src/data/tie_break.py

```diff
@@ -167,8 +169,10 @@ def adjusted_score(
                 ):
                     score += Result.DRAW.points(tournament.point_values)
                 else:
+                    assert pairing.result is not None
                     score += pairing.result.points(tournament.point_values)
             else:
+                assert pairing.result is not None
```

Comment:
I think it should be a saparate case:
if `pairing.result` is `None`, ignore this `Pairing` and continue

Reply:
This PR is just for type checking.  I shall remove the assertion and the code can be revisited later.